### ISSUE TITLE
NEW SCROBBLER: https://github.com/Taxel/PlexTraktSync

### ISSUE THREAD

**Issue by snickers2k** [trakt/Plex-Trakt-Scrobbler#620]
**i recommend to use this new scrobbler, actively maintained** 
(since this one here doesn't work for me anymore)

**https://github.com/Taxel/PlexTraktSync**

**Comment by snickers2k** [comment]
please add that one to the readme @rg9400 or @fuzeman 
thanks

**Comment by the-jeffski** [comment]
This is misleading as the new project is not a 'SCROBBLER' .  It just does a sync on a timer. So if you want to track as you watch it is not for that.

**Comment by ourtv** [comment]
> This is misleading as the new project is not a 'SCROBBLER' . It just does a sync on a timer. So if you want to track as you watch it is not for that.

Somebody didn't read the whole thing....

about halfway down the page my friend...


"Watch

You can use the watch command to listen to events from Plex Media Server and scrobble plays......"

**Comment by the-jeffski** [comment]
> > This is misleading as the new project is not a 'SCROBBLER' . It just does a sync on a timer. So if you want to track as you watch it is not for that.
> 
> Somebody didn't read the whole thing....
> 
> about halfway down the page my friend...
> 
> "Watch
> 
> You can use the watch command to listen to events from Plex Media Server and scrobble plays......"

Ah, you are right. I'd not scrolled right down to the bottom after all the other sections. It needs a better readme so that its obvious and not hidden.

**Comment by LostOnTheLine** [comment]
> > This is misleading as the new project is not a 'SCROBBLER' . It just does a sync on a timer. So if you want to track as you watch it is not for that.
> 
> Somebody didn't read the whole thing....
> 
> about halfway down the page my friend...
> 
> "Watch
> 
> You can use the watch command to listen to events from Plex Media Server and scrobble plays......"

Yeah, but that's still far from the old one. With the old one that uses webhooks you literally could open up Trakt & it'd tell you that you are currently paused at 11:14 of episode 14 of "The Elephant's Green Shoes" or whatever. This is still just syncing it on a schedule, which is what the complaint was about, & not **_Actively_** scrobbling it, just _**Passively**_ doing it

**Comment by ourtv** [comment]
You're not wrong. It also does not sync how much you have watched.

**Comment by jeff15110168** [comment]
Can anyone comment on how PlexTraktSync compares to Plex-Trakt-Scrobbler? I am having issues once again with the latter not properly scrobbling, and trying to figure out what to do (make a switch to PlexTraktSync or again try to troubleshoot...)
